helper/schema/schema.go

```diff
@@ -131,6 +131,9 @@ type Schema struct {
 	// This string is the message shown to the user with instructions on
 	// what do to about the removed attribute.
 	Removed string
+
+	// ValidateFunc allows individual fields to validate
```

Comment:
Yeah that was my intent, I just never finished the docstring. :see_no_evil: Done now.